include/galois.h

```diff
@@ -46,57 +46,69 @@
 extern "C" {
 #endif
 
-extern int galois_init_default_field(int w);
-extern int galois_uninit_field(int w);
-extern void galois_change_technique(gf_t *gf, int w);
+typedef struct gf2_t_ {
+	gf_t* gf;
+	gf_t* gfp_array[64];
+	int gfp_is_composite[64];
+} gf2_t;
```

Comment:
I really don't recommend naming this struct as `gf2_t` because it is easily confused with `GF(2)`, the finite field with two elements. (`gf_state_t` should be better I think?)